### ISSUE TITLE
Dagger/Autolykos trex dual miner not showing

### ISSUE THREAD

**Issue by AKiman87** [angelbbs/NiceHashMinerLegacy#142]
I have 2 3080 ti's on two different rigs. Both do now show the daggerhashimoto/Autolykos mining algorithm in the device algorithm pain. On one of the rigs I have a 3060 Ti that does show the algorithm as an option. The algorithm should show as an option for the 3080 ti.

**Comment by angelbbs** [comment]
Dual algos support only NVIDIA LHR GPU
Available combinations along with memory requirements:
DaggerAutolykos (8GB+)
DaggerKAWPOW (10GB+)
DaggerOctopus (10GB+)

**Comment by AKiman87** [comment]
Yes, so the 2 3080 ti's I have should be ablt to run all three. All Nvidia 3080 ti's are LHR cards, with 12 gb of vram.  Only Dagger Kawpow and Dagger Octo show up on my available algorithm list. I have verified that Dagger Autolyko is in the trex file.

Edit,
Thanks for the quick response.

**Comment by angelbbs** [comment]
Send file logs/log.txt after starts program

**Comment by AKiman87** [comment]
[log.txt](https://github.com/angelbbs/NiceHashMinerLegacy/files/7715766/log.txt)
[log.txt](https://github.com/angelbbs/NiceHashMinerLegacy/files/7715775/log.txt)
